analiza-rozwiazan-biblioteki-komponentow-GUI.rst

```diff
@@ -0,0 +1,330 @@
+Analiza rozwiązań w zakresie bibliotek komponentów GUI
+======================================================
+
+Streszczenie
+------------
+
+Cele
+~~~~
+
+1. Zbadanie istniejących bibliotek GUI pod kątem technicznym i
+   biznesowym.
+2. Określenie strategii rozwoju biblioteki GUI.
+3. Uszczegółowienie kroków, które zostaną zrealizowane w drugiej fazie
+   projektu „projektowania biblioteki GUI”.
+
+Sposób realizacji
+~~~~~~~~~~~~~~~~~
+
+Pierwszym etapem projektu była faza badawcza, w trakcie której zostały
+opracowane merytoryczne wytyczne do dalszych działań projektowych.
+Proces badawczy trwał 3 tygodnie i był realizowany przez zespoły UX oraz
+IT.
+
+Analiza była prowadzona równolegle na dwóch poziomach:
+
+1. poziomie technicznym – przeprowadzono analizę istniejących
+   frameworków frontendowych uwzględniającą następujące parametry:
+
+   1. Główne zastosowanie frameworku
+   2. Kontekst mobilny (łatwość realizowania aplikacji w technologii
+      RWD)
+   3. Open source (otwarta licencja bez ograniczeń)
```

Comment:
Wynikało to z chęci podkreślenia charakteru licencji. Nie wszystkie są równe pod względem zasad otwartości. @trueskawka czy chcemy to poprawić?